src/main/java/duke/utils/Storage.java

```diff
@@ -0,0 +1,203 @@
+
+package duke.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import duke.entities.Deadline;
+import duke.entities.Event;
+import duke.entities.Task;
+import duke.entities.Todo;
+import duke.enums.Regex;
+import duke.exceptions.DukeException;
+
+/**
+ * Handles writing into storage and reading from storage
+ */
+public class Storage {
+    private static Pattern loadRegex = Pattern.compile(Regex.LOADING_REGEX_WITH_DEADLINE.toString());
+    private static final HashMap<String, String> monthToInt = new HashMap<String, String>() {
+        {
+            put("January", "01");
+            put("February", "02");
+            put("March", "03");
+            put("April", "04");
+            put("May", "05");
+            put("June", "06");
+            put("July", "07");
+            put("August", "08");
+            put("December", "12");
+            put("November", "11");
+            put("October", "10");
+            put("September", "09");
```

Comment:
I think can be clearer if named in order of the month, with December at the end 😄